server/lib/tuist_web/live/build_run_live.ex

```diff
@@ -27,7 +27,7 @@ defmodule TuistWeb.BuildRunLive do
 
     run =
       run
-      |> Tuist.Repo.preload([:project, :ran_by_account])
+      |> Tuist.Repo.preload([:project, :ran_by_account, project: :vcs_connection])
```

Comment:
Should we preload this by default in the plug that loads the project? I've seen these preloads scattered in this PR.